src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,127 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
   allowfullscreen
 ></iframe>
 
+
+## Step-by-step guide
+
+If you want to know the whole creation process in detail, follow the instructions we’ve prepared. We'll use an app named Your Guide as our example.
+
+### **Step 1 Getting started**
+
+Make sure you are signed in to the [Developer Console](https://developers.livechat.com/console/). Your LiveChat credentials should allow you to be logged in automatically. If not, you can [sign up here](https://accounts.livechat.com/signup?client_id=bb9e5b2f1ab480e4a715977b7b1b4279&redirect_uri=https%3A%2F%2Fmy.livechatinc.com&response_type=token&source_id=create_account_link&source_url=https%3A%2F%2Faccounts.livechat.com&source_type=accounts).
+
+Now, we go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. You can always change your name later.
+
+💡 **Need helping hand?** Right after signing in to the Developers Console, you can launch the **App wizard**. It will guide you through the whole app creation process.
```

Comment:
```suggestion
💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App wizard**. It will guide you through the whole app creation process.
```